tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -994,9 +994,12 @@ public string IgnoreErrorsAndIgnoreClassesCombineTests(string[] ignoreClasses, s
             return string.Join(",", _defaultConfig.IgnoreErrorsConfiguration.Keys);
         }
 
-        [TestCase("401", "405", ExpectedResult = "405")]
-        [TestCase("401", null, ExpectedResult = "401")]
-        public string ExpectedStatusCodesSetFromLocalAndServerOverrides(string local, string server)
+
+
+        [TestCase("401", new[] { "405" }, ExpectedResult = new[] { "405" })]
+        [TestCase("401", new string[0], ExpectedResult = new string[0])]
+        [TestCase("401", null, ExpectedResult = new[] { "401" })]
+        public IEnumerable<object> ExpectedStatusCodesSetFromLocalAndServerOverrides(string local, string[] server)
```

Comment:
This should be object[] as part of the input allowing server to send back either ints or strings?
Please add test 
for server new[]{400}
for server new[]{400,"401"}

Reply:
I see the test below, maybe combine them?

Reply:
this test is testing server config overriding behavior after deserialization finishes so the value for _serverConfig.RpmConfig.ErrorCollectorExpectedStatusCodes can only be IEnumerable<string>. I need to fix the the return type for the test to be IEnumerable<string> instead of IEnumerable<object> due to recent changes.

Reply:
ok